### ISSUE TITLE
Build broken on Debian testing (missging widl binary)

### ISSUE THREAD

**Issue by shmerl** [HansKristian-Work/vkd3d-proton#1450]
Commit 2ec6c90ab6676e328883651783e3822579234386 broke building vkd3d-proton on Debian (testing).

I think the only source of widl there is `mingw-w64-tools` package which provides:

```
/.
/usr
/usr/bin
...
/usr/bin/i686-w64-mingw32-widl
...
/usr/bin/x86_64-w64-mingw32-widl
```

So there is no simply `widl` binary.

**Comment by shmerl** [comment]
As a workaround, I created a symlink in `$HOME/.local/bin`:

```
ln -s /usr/bin/x86_64-w64-mingw32-widl widl
```

**Comment by shmerl** [comment]
I see there is also this from `wine64-tools`:  `/usr/bin/widl-stable`.

**Comment by shmerl** [comment]
Hm, that's actually a newer one. So doing this also helped:

```
ln -s /usr/bin/widl-stable $HOME/.local/bin/widl
```

**Comment by shmerl** [comment]
Btw, I made a diff with the patches: https://pastebin.com/raw/ijZkZdDg

I noticed it has something unrelated to fsync, like some OpenCL stuff? Looks like even in mainline preset, it applies something more, unless I'm missing something.

**Comment by HansKristian-Work** [comment]
> Hm, that's actually a newer one. So doing this also helped:
> 
> ```
> ln -s /usr/bin/widl-stable $HOME/.local/bin/widl
> ```

Why is Debian renaming the widl binary like this?

**Comment by shmerl** [comment]
Not sure, but my suspicion is that may be they wanted to differentiate wine64-tools as "stable" and "development" variants?

That's what I see in it:

```
...
/usr/bin/widl-stable
/usr/bin/winebuild-stable
/usr/bin/winecpp-stable
/usr/bin/winedump-stable
/usr/bin/wineg++-stable
/usr/bin/winemaker-stable
/usr/bin/wmc-stable
/usr/bin/wrc-stable
...
```

**Comment by shmerl** [comment]
May be one option would be to check for widl-stable in addition or to mention in documentation that symlink would be needed on Debian.

**Comment by r2rX** [comment]
After a few weeks of not updating vkd3d-proton for Lutris, I tried compiling it today but it seems that a similar issue is present with Fedora 37 as well. 

All dependencies are installed (as I was compiling fine earlier), running Meson v1.0.0, WINE v8.2 and mingw-w64-tools v10.0.0 from Fedora's repo.

When searching for 'widl', my system finds the following:

```
/usr/bin/i686-w64-mingw32-widl
/usr/bin/x86_64-w64-mingw32-widl
/usr/bin/x86_64-w64-mingw32ucrt-widl
```

I did exclude a few entries from the list above, specifically for WINE runners in Lutris, Bottles and Heroic (which include 'widl' in their respective 'bin' directories) but I didn't notice one for the system-wide installed version of WINE from Fedora's repo, aside from what was listed above.

Not sure if this makes a difference (although, as mentioned, it was compiling fine a few weeks ago on the same system).

[meson-log.txt](https://github.com/HansKristian-Work/vkd3d-proton/files/10989619/meson-log.txt)
